src/mainviews/create-job-from-definition.tsx

```diff
@@ -139,6 +139,7 @@ export function CreateJobFromDefinition(
         createJobFromDefinitionModel
       )
       .then(response => {
+        log('create-job-from-definition.create-job.success');
```

Comment:
can we log the error response - http response code and message ?

Reply:
Added optional `error` parameter with error message and HTTP status code to log object's `body` : https://github.com/jupyter-server/jupyter-scheduler/pull/472/files#diff-0b5adbfe7b36e4ae2f479291e20152e33e940f7f265162d77f40f6bdb5da7405R44-R47